### ISSUE TITLE
Training process related to misleading information

### ISSUE THREAD

**Issue by kevinchiu19** [opendilab/LMDrive#16]
Hello, thank you for your wonderful work!

I have a question. How is the training setting related to misleading instructions mentioned in the paper implemented? I don’t seem to see it in the code.

Looking forward to your reply, thank you~

**Comment by deepcs233** [comment]
Hi!

In the project, we added some labeled misleading data into the training dataset to let the model recognize the misleading instructions.  We will release the corresponding data soon!

**Comment by kevinchiu19** [comment]
Thanks for your quick reply.

I would like to confirm with you, which part of the code corresponds to "To enable the model to reject misleading instructions, we label the corresponding data as ‘completed’ after the misleading instruction is given for about 1 second." mentioned in the paper.

**Comment by deepcs233** [comment]
Hi!
Yes, we haven't released the corresponding code and dataset. However, the current code and dataset can also train an agent that has limited ability to reject misleading instructions. In our paper, the agent rejects the misleading instructions by recognizing the current `misleading` instruction should be finished in a very short time.

**Comment by kevinchiu19** [comment]
Okay, I understand, thank you for your patient reply.
